uiService/api.go

```diff
@@ -133,47 +133,46 @@ func resetToDefaultUIHandler(service *UIService) func(http.ResponseWriter, *http
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
-		if UIVersion(currentVersion) == PreBundledUIVersion {
-			w.Header().Add("Content-Type", "text/plain")
-			w.WriteHeader(http.StatusOK)
-			w.Write([]byte("OK"))
-			return
-		}
-		logrus.WithField("CurrentVersion", currentVersion).Debug("Received reset request.")
 
-		if updatingVersion, lockErr := setServiceUpdating(service, ""); lockErr != nil {
-			var message string
-			if UIVersion(updatingVersion) == PreBundledUIVersion {
-				message = "Cannot process reset, another reset is currently in progress."
-			} else {
-				message = "Cannot process reset, an update is currently in progress."
+		if UIVersion(currentVersion) != PreBundledUIVersion {
+			logrus.WithField("CurrentVersion", currentVersion).Debug("Received reset request.")
```

Comment:
We should move this through line 154 `defer resetServiceFromUpdate(service)` to above and outside this `if`. We would want the service set to updating while we clear the versions root.